test/org/sagebionetworks/bridge/dynamodb/DynamoTaskDaoMockTest.java

```diff
@@ -120,8 +123,8 @@ public void testOfFirstPeriod() throws Exception {
         assertTask("2015-04-12T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(0));
         assertTask("2015-04-13T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(1));
         assertTask("tapTest", TestConstants.ACTIVITY_3_REF, tasks2.get(2));
-        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(3));
-        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_1_REF, tasks2.get(4));
+        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_1_REF, tasks2.get(3));
+        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(4));
```

Comment:
Added notes, also re-arranged the arguments and fixed them so tapTest was in the same position as the diatomite string, which was confusing.